src/import/sb2specmap.js

```diff
@@ -22,1367 +22,1367 @@
     },
-    'playSound:':{
-        'opcode':'sound_play',
-        'argMap':[
+    'playSound:': {
+        'opcode': 'sound_play',
+        'argMap': [
             {
-                'type':'input',
-                'inputOp':'sound_sounds_option',
-                'inputName':'SOUND_MENU'
+                'type': 'input',
+                'inputOp': 'sound_sounds_option',
+                'inputName': 'SOUND_MENU'
             }
         ]
     },
-    'doPlaySoundAndWait':{
-        'opcode':'sound_playuntildone',
-        'argMap':[
+    'doPlaySoundAndWait': {
+        'opcode': 'sound_playuntildone',
+        'argMap': [
             {
-                'type':'input',
-                'inputOp':'sound_sounds_option',
-                'inputName':'SOUND_MENU'
+                'type': 'input',
+                'inputOp': 'sound_sounds_option',
+                'inputName': 'SOUND_MENU'
             }
         ]
     },
-    'stopAllSounds':{
-        'opcode':'sound_stopallsounds',
-        'argMap':[
+    'stopAllSounds': {
+        'opcode': 'sound_stopallsounds',
+        'argMap': [
         ]
     },
-    'playDrum':{
-        'opcode':'sound_playdrumforbeats',
-        'argMap':[
+    'playDrum': {
+        'opcode': 'sound_playdrumforbeats',
+        'argMap': [
             {
-                'type':'input',
-                'inputOp':'math_number',
-                'inputName':'DRUMTYPE'
+                'type': 'input',
+                'inputOp': 'math_number',
+                'inputName': 'DRUMTYPE'
             },
             {
-                'type':'input',
-                'inputOp':'math_number',
-                'inputName':'BEATS'
+                'type': 'input',
+                'inputOp': 'math_number',
+                'inputName': 'BEATS'
             }
         ]
     },
-    'rest:elapsed:from:':{
-        'opcode':'sound_restforbeats',
-        'argMap':[
+    'rest:elapsed:from:': {
+        'opcode': 'sound_restforbeats',
+        'argMap': [
             {
-                'type':'input',
-                'inputOp':'math_number',
-                'inputName':'BEATS'
+                'type': 'input',
+                'inputOp': 'math_number',
+                'inputName': 'BEATS'
             }
         ]
     },
-    'noteOn:duration:elapsed:from:':{
-        'opcode':'sound_playnoteforbeats',
-        'argMap':[
+    'noteOn:duration:elapsed:from:': {
+        'opcode': 'sound_playnoteforbeats',
+        'argMap': [
             {
-                'type':'input',
-                'inputOp':'math_number',
-                'inputName':'NOTE'
+                'type': 'input',
+                'inputOp': 'math_number',
+                'inputName': 'NOTE'
             },
             {
-                'type':'input',
-                'inputOp':'math_number',
-                'inputName':'BEATS'
+                'type': 'input',
+                'inputOp': 'math_number',
+                'inputName': 'BEATS'
```

Comment:
Yes, it's kind of cool — you run `npm run lint -- --fix`! Most whitespace-related rules can be automatically resolved.